vignettes/wwinference.Rmd

```diff
@@ -382,9 +384,26 @@ draws_df <- get_draws(ww_fit)
 print(draws_df)
 ```
 
+Note that the `get_draws_df()` function will return a tidy long
+dataframe with all of the posterior draws for the calibration, nowcast, and
+forecast period joined to applicable data for each of
+the included variables in the `name` column:
+  - counts(`"predicted counts"`)
+  - site-level wastewater concentrations (`"predicted wastewater"`)
+  - "global"(e.g. state) R(t) estimate (`"global R(t)"`)
+  - "local" (site + the one auxiliary subpopulation) R(t) estimates (`"subpopulation R(t)"`)
+
+If only interested in the hospital admissions estimates
+(`"predicted counts"`) or the wastewater estimates (`"predicted wastewater"`),
+we suggest immediately filtering to `name %in% c({your_desired_outputs})`
+so as not to carry around extra data.
+This is particularly important if there are a large
+number of wastewater sites, as this dataframe can get quite large.
+
```

Comment:
```suggestion
```
I believe the section immediately below is correct as of #153 and so this should be deleted

Reply:
Whoops I thought I did delete this.

Reply:
Oh no I fixed this in the other PR and I am getting them all confused..